packages/store/src/services/store/operations/store-import.ts

```diff
@@ -85,24 +85,36 @@
     importUrl: string,
     flags: FlagOptions,
   ): Promise<BulkDataOperationByIdResponse> {
-    const importResponse: BulkDataStoreImportStartResponse = await this.apiClient.startBulkDataStoreImport(
-      apiShopId,
-      targetShopDomain,
-      importUrl,
-      parseResourceConfigFlags(flags.key as string[]),
-      bpSession,
-    )
+    try {
+      const importResponse: BulkDataStoreImportStartResponse = await this.apiClient.startBulkDataStoreImport(
+        apiShopId,
+        targetShopDomain,
+        importUrl,
+        parseResourceConfigFlags(flags.key as string[]),
+        bpSession,
+      )
+
+      if (!importResponse.bulkDataStoreImportStart.success) {
+        const errors = importResponse.bulkDataStoreImportStart.userErrors.map((error) => error.message).join(', ')
+        throw new OperationError('import', ErrorCodes.BULK_OPERATION_FAILED, {
+          errors,
+          operationType: 'import',
+        })
+      }
 
-    if (!importResponse.bulkDataStoreImportStart.success) {
-      const errors = importResponse.bulkDataStoreImportStart.userErrors.map((error) => error.message).join(', ')
-      throw new OperationError('import', ErrorCodes.BULK_OPERATION_FAILED, {
-        errors,
-        operationType: 'import',
-      })
+      const operationId = importResponse.bulkDataStoreImportStart.operation.id
+      return this.apiClient.pollBulkDataOperation(apiShopId, operationId, bpSession)
+    } catch (error) {
```

Comment:
Could this catch statement be a part of `startBulkDataStoreImport`? I think it might fir better to API client.